etc/redfishpower-cray-r272z30.dev

```diff
@@ -10,12 +10,36 @@
 #
 # - This device specification was tested on a Cray with Gigabyte R272-Z30.
 #
-# - If using a slightly different model or BMC firmware version, its
-#   possible the URI paths will be different.  A good starting point
-#   is to run the following command to discover the correct paths:
+# - CAUTION: If you intend to use this file as the basis for a different
+#   Redfish system, please note the following.
+#
```

Comment:
One thought to avoid the repetition would be to add a section to the redfishpower(8) man page and in each dev file just replace the whole block with a pointer to the man page?

Reply:
good idea, i forgot I had a redfishpower manpage :-)